### DISCUSSION TITLE
[Profiler] Get current request as cURL

### DISCUSSION THREAD

**Discussion by NikitaKharkov** [symfony/symfony#53831] [Q&A]
Hi, Symfoniers! :)
Is there any way to get a cURL of the current request in the request profiler?

I didn't find any in the request/response tab. Maybe in another place or another Symfony version? My is 6.4.

Yes, I understand that I can get it from the tab network in chrome dev tools, but in it's much easier to get it in the Profiler tool.

If not - I'd like to open a suggestion issue to implement it.

**Comment by stof** [comment]
There is no such feature in the profiler (we have such a feature for the HttpClient panel, where we are on the sending end of the request rather than the receiving end).
